README.adoc

```diff
@@ -52,7 +52,11 @@ http://maven.apache.org/plugins/maven-deploy-plugin/examples/deploy-ssh-external
 
 You can alternatively use a docker container to build the website :
 
-    $ docker build -t james/site dockerfiles/site
+You need to build the homepage by:
```

Comment:
My problem is:
- Java6 is installed on docker debian wheezy. Jekyll need ruby which is installed on docker debian Jessi. So if we merge them all in one we need:
1. Find the way to install java6 on Jessie docker with had ruby: I tried but I did not success
2. Find the way to install ruby on Wheezy docker which had java6: it worked but team said it's quite complex because we don't want to maintain a docker which installed ruby.